main/guides/getting-started/README.md

```diff
@@ -27,8 +27,8 @@ If `agoric` is not found, then you may need to add the parent directory of where
 This presentation is a good overview of the Agoric SDK setup process,
 though a few details are out of date:
 
-- node version: 12.x is too old; use 14.15.0 or higher
-- skip `git checkout hackathon-2020-11`; use the default `master` branch
+- node version: 12.x is too old; use 16.19.1 or higher
+- skip `git checkout hackathon-2020-11`; use the `community-dev` branch
 
```

Comment:
Have you verified that Agoric now requires this later version of Node?

Reply:
Yes. This is the requirement in agoric-sdk. See https://github.com/Agoric/agoric-sdk/blob/master/package.json#L36